packages/familie-backend/src/headers.ts

```diff
@@ -5,8 +5,10 @@ const fontSource = 'https://fonts.gstatic.com';
 const navFontSource = 'https://cdn.nav.no';
 const amplitude = 'https://amplitude.nav.no';
 const sentry = 'https://sentry.gc.nav.no';
+const navTelemetry = 'https://telemetry.nav.no';
+const navTelemetryDev = 'https://telemetry.ekstern.dev.nav.no';
 
-const cspString = `default-src 'self' data: ${amplitude} ${sentry}; style-src 'self' ${styleSource} data: 'unsafe-inline'; font-src 'self' ${fontSource} ${navFontSource} data:; frame-src 'self' blob:;`;
+const cspString = `default-src 'self' data: ${amplitude} ${sentry} ${navTelemetry} ${navTelemetryDev}; style-src 'self' ${styleSource} data: 'unsafe-inline'; font-src 'self' ${fontSource} ${navFontSource} data:; frame-src 'self' blob:;`;
```

Comment:
Er ikke helt heldig å stappe inn begge to hver gang, men kom ikke frem til en god måte å finne nåværende miljø når det er et fellesbibliotek. Åpen for innspill

Reply:
Er enig, dette er enkleste løsning, som sannsynligvis er beste løsning ☺️